{
  "name": "fastify-dd-trace-bug",
  "version": "3.0.0",
  "description": "fastify3 getting corrupted by dd-trace",
  "scripts": {
    "start": "yarn dev",
    "dev": "cross-env NODE_ENV=development nodemon",
    "debug": "nodemon --config nodemon-debug.json",
    "format": "prettier --write \"src/**/*.ts\"",
    "build": "rimraf dist && tsc",
    "prod": "cross-env NODE_ENV=production node dist/index.js"
  },
  "author": "Rainer Burgstaller <rburgst@gmail.com>",
  "license": "MIT",
  "dependencies": {
    "dd-trace": "^0.24.1",
    "fastify": "^3.2.1"
  },
  "devDependencies": {
    "@types/node": "^14.6.0",
    "cross-env": "^7.0.2",
    "nodemon": "^2.0.4",
    "prettier": "^2.0.5",
    "rimraf": "^3.0.2",
    "ts-node": "^8.10.2",
    "tsconfig-paths": "^3.9.0",
    "typescript": "^3.9.7"
  }
}
